scripts/sem_bleaching_map.py

```diff
@@ -714,7 +714,7 @@ def save_data(self, data, fn):
         else:
             logging.info("Saving file '%s'", fn)
 
-        exporter.export(unicode(fn), data)
+        exporter.export(str(fn), data)
```

Comment:
You can pass `fn` directly. It's supposed to be a string already. 
Ideally, you'd also update the docstring, to just indicate `fn` is a `str`.